simulations/run_agent_info_job.py

```
@@ -0,0 +1,37 @@
+    os.environ["EPISTEMIX_API_URL"] = "http://localhost:5555"
+
+    print("Starting agent_info_job execution...")
+    print(f"API URL: {os.environ.get('EPISTEMIX_API_URL', 'Not set')}")
+    print(f"Job tags: {info_job.tags}")
+    print(f"FRED files: {info_job.fred_files}")
+
+    try:
+        # Execute the job with a timeout of 300 seconds
+        info_job.execute(300)
+        print(f"Job completed successfully! Status: {info_job.status}")
+        return 0
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)